test/server/apis/entry-api_test.js

```diff
@@ -36,29 +36,25 @@ describe('The Entries API', () => {
   describe('DELETE /entries', () => {
     it_('deletes an entry', function * deletesLinks() {
       modelStub = sinon.stub(Entry, 'remove');
-      modelStub.resolves(data.id);
+      modelStub.resolves({ success: true });
       yield request(app)
         .delete('/api/yt/entries/1')
         .expect(200)
         .expect(response => {
-          expect(response.body.entry).to.equal(data.id);
-          expect(Entry.remove.calledOnce).to.equal(true);
-          expect(Entry.remove.calledWith('1')).to.equal(true);
+          expect(response.body.data.success).to.equal(true);
         });
     });
   });
 
   describe('PUT /entries', () => {
     it_('updates an entry', function * updateLink() {
-      modelStub = sinon.stub(Entry, 'update');
+      modelStub = sinon.stub(Entry, 'updateByID');
       modelStub.resolves(data);
       yield request(app)
         .put('/api/yt/entries/1')
         .expect(200)
         .expect(response => {
-          expect(response.body.entry).to.include(data);
-          expect(Entry.update.calledOnce).to.equal(true);
-          expect(Entry.update.calledWith('1')).to.equal(true);
```

Comment:
We probably still want this line to verify that the right things are being called in the model. 

Because we're stubbing, without this assertion, we won't have anyway to know that